lib/rubocop/cop/rails/compact_blank.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks for places where custom logic on rejection blanks
+      # from arrays and hashes can be replaced with `compact_blank`.
+      #
+      # @example
+      #
+      #   # bad
+      #   collection.reject { |e| e.blank? }
```

Comment:
To me, this is not sufficiently bad. Especially with block-pass:
```ruby
collection.reject(&:blank?)
```
However, with hashes this becomes unwieldy, and I don't think there's a better option than `compact_blank`:
```ruby
hash_collection.reject { |_k, v| v.blank? }
```

Even though I'd rather prefer having something more generic like:
```ruby
hash_collection.reject_values(&:blank?)
```